components/eventing-controller/pkg/env/nats_config.go

```diff
@@ -48,6 +48,17 @@ type NatsConfig struct {
 	// Prefix for the JetStream stream subjects filter.
 	// It will be overridden by non-empty EventTypePrefix.
 	JSStreamSubjectPrefix string `envconfig:"JS_STREAM_SUBJECT_PREFIX" required:"true"`
+
+	// Deliver policy determines for a consumer where in the stream it wants to start receiving messages
```

Comment:
`Deliver policy` is an awful word, but it's what NATS called it so I have to accept it. I'd format it in upper case so it's clear that it's a proper name.

```suggestion
	// Deliver Policy determines for a consumer where in the stream it starts receiving messages
```

I don't think we need the "want" ?

Reply:
Done.